cmd/gh2gcs/cmd/root.go

```diff
@@ -198,11 +207,17 @@ func run(opts *options) error {
 			return errors.Wrap(err, "downloading release assets")
 		}
 
-		if err := gh2gcs.Upload(&rc, gh, opts.outputDir); err != nil {
-			return errors.Wrap(err, "uploading release assets to GCS")
+		if !opts.downloadOnly {
+			if err := gh2gcs.Upload(&rc, gh, opts.outputDir); err != nil {
+				return errors.Wrap(err, "uploading release assets to GCS")
+			}
 		}
 	}
 
+	if opts.downloadOnly {
+		logrus.Infof("Files downloaded to %s folder", opts.outputDir)
```

Comment:
We probably can log this in any case. I would just move it directly after the call to `DownloadReleases`

Reply:
+1

Reply:
done! thanks for the review!